note-well.md

```diff
@@ -0,0 +1,25 @@
+# IETF Note Well 
```

Comment:
We should avoid replicating the note well in every WG repository.  The relevant links are already in `CONTRIBUTING.md`.  If you want to update that to be more like the official Note Well (is there one?), please make a separate PR that updates that file instead of adding a new one.  But this seems pretty far afield from the RPC's role?